localization/english/modifiers/wc_modifiers_l_english.yml

```diff
@@ -7,25 +7,34 @@
  light_aerial_screen_mult:0 "$light_aerial_screen_add$"
  light_aerial_toughness_add:1 "[light_aerial|E] [toughness|E]"
  light_aerial_toughness_mult:0 "$light_aerial_toughness_add$"
+ light_aerial_siege_value_add:1 "[light_aerial|E] [siege_progress|E]"
  
  massive_aerial_damage_add:1 "[massive_aerial|E] [damage|E]"
  massive_aerial_damage_mult:0 "$massive_aerial_damage_add$"
  massive_aerial_pursuit_add:1 "[massive_aerial|E] [pursuit|E]"
  massive_aerial_pursuit_mult:0 "$massive_aerial_pursuit_add$"
  massive_aerial_screen_add:1 "[massive_aerial|E] [screen|E]"
+ massive_aerial_screen_mult:0 "$massive_aerial_screen_add$"
  massive_aerial_toughness_add:1 "[massive_aerial|E] [toughness|E]"
  massive_aerial_toughness_mult:0 "$massive_aerial_toughness_add$"
+ massive_aerial_siege_value_add:1 "[massive_aerial|E] [siege_progress|E]"
  
  caster_damage_add:1 "[caster|E] [damage|E]"
  caster_damage_mult:0 "$caster_damage_add$"
  caster_toughness_add:1 "[caster|E] [toughness|E]"
  caster_toughness_mult:0 "$caster_toughness_add$"
+ caster_pursuit_add:1 "[caster|E] [pursuit|E]"
+ caster_pursuit_mult:0 "$caster_pursuit_add$"
  caster_screen_add:1 "[caster|E] [screen|E]"
+ caster_screen_mult:0 "$caster_screen_add$"
  caster_siege_value_add:1 "[caster|E] [siege_progress|E]"
  
- massive_infantry_siege_value_add:1 "[massive_infantry|E] [siege_progress|E]"
  massive_infantry_damage_add:1 "[massive_infantry|E] [damage|E]"
  massive_infantry_damage_mult:0 "$massive_infantry_damage_add$"
  massive_infantry_toughness_add:1 "[massive_infantry|E] [toughness|E]"
  massive_infantry_toughness_mult:0 "$massive_infantry_toughness_add$"
+ massive_infantry_pursuit_add:1 "[massive_infantry|E] [pursuit|E]"
+ massive_infantry_pursuit_mult:0 "$massive_infantry_pursuit_add$"
  massive_infantry_screen_add:1 "[massive_infantry|E] [screen|E]"
+ massive_infantry_screen_mult:0 "$massive_infantry_screen_add$"
+ massive_infantry_siege_value_add:1 "[massive_infantry|E] [siege_progress|E]"
```

Comment:
You should avoid inserting lines in between others, new lines are supposed to always be added at the end.

Reply:
Dont worry, I made this file in my last branch and I'm okay with this layout.  Looks like I missed some modifier localization and I added it in its appropriate spot.